plugins/soundsourcemediafoundation/soundsourcemediafoundation.cpp

```diff
@@ -478,7 +478,16 @@ bool SoundSourceMediaFoundation::configureAudioStream(const AudioSourceConfig& a
                 << "failed to get current media type from stream";
         return false;
     }
+    //------------ Get bitrate from the file, before we change it to get uncompressed audio
+    UINT32 aacbitrate;
```

Comment:
Naming: "avgBytesPerSecond"

The actual "bit"rate is calculated below and this value is not tied to AAC.